lovr-mouse.lua

```diff
@@ -1,3 +1,6 @@
+-- This should not even attempt to run if we are not using LuaJIT (no FFI) or if we are on Android (no GLFW)
+if type(jit) ~= 'table' or lovr.headset.getDriver() == "oculusmobile" then return false end
```

Comment:
2 problems
- I *THINK* returning false is idiomatic, but I'm not… sure.
This means you can do
lovr.mouse = require "lovr-mouse"
and then later do
if lovr.mouse then
etc, but it does mean if you do
lovr.mouse.isDown()
without checking, you will fail in an ugly way.
* **bigger problem: is it supported to disable lovr.headset?** if you disable lovr.headset then == oculusmobile will crash… but i don't know how to check "are we on android" without lovr.headset.